pkg/kubecost/allocation.go

```diff
@@ -284,6 +306,26 @@ func (a *Allocation) RAMTotalCost() float64 {
 	return a.RAMCost + a.RAMCostAdjustment
 }
 
+func (a *Allocation) PVTotalCost() float64 {
+	return a.PVCost() + a.PVCostAdjustment
+}
+
+func (a *Allocation) PVCost() float64 {
+	cost := 0.0
+	for _, pv := range a.PVBreakdown {
+		cost += pv.Cost
+	}
+	return cost
+}
+
+func (a *Allocation) PVByteHours() float64 {
+	byteHours := 0.0
+	for _, pv := range a.PVBreakdown {
+		byteHours += pv.ByteHours
+	}
+	return byteHours
+}
```

Comment:
Cool cool. I like having these as function. Do you mind commenting them, as they are public?

If your IDE doesn't warn you about un-commented exported functions, we can try to set that up, too. Everyone who uses VSCode with the Go plugin will get warned about this, so it'd be good to get parity across everyone's IDEs.

Reply:
I turned it on and cranked the warning level up to error
![Screen Shot 2021-05-06 at 5 54 09 PM](https://user-images.githubusercontent.com/12225425/117382830-3c993b00-ae94-11eb-89a8-6c1f784892f1.png)